### ISSUE TITLE
Final Fantasy VI, older remake. (Plus others)

### ISSUE THREAD

**Issue by DaftOtter** [vita-nuova/bounties#98]
Hello there,

I’m just trying to see if final fantasy vi is portable to vita. I’m not talking about the pixel remasters, I’m aware they aren’t possible right now, I’m talking about the previous remake, that is no longer available. The one with over done sprites.

If someone who knows their stuff could let me know that it can be done, I’ll save up for a while and put a bounty on it. 

Also, I will be putting a bounty on the final fantasy v android to vita port to be updated. I absolutely love it, but it crashes often, it has several bugs. I’ve got over 30 hours on it though, it’s amazing. I’m not sure when I can afford it, I’ll need to save a bit so the amount is reasonable. 

Also, one last thing, does anyone know if the final fantasy vii, viii and ix remasters, the version where it’s basically the same except for 3 cheats on all of them are portable. 

Basically, what I’ll do if someone can confirm these all are possible, is put a bounty on one at a time starting with v being updated then going through them in order one at a time.

Thanks for your time, and all the great vita awesomeness you’ve brought to my life,

Sam

**Comment by Rinnegatamante** [comment]
Final Fantasy VI and VII are portable. VIII and IX are Unity so VERY UNLIKELY for someone to port them (unless a very huge bounty gets set for them).

**Comment by frangarcj** [comment]
I'm going to spend a little time working on FFV and try to fix the bugs and then FF Dimensions.

**Comment by Smartplus50** [comment]
Damn Nice! I might be willing to contribute (a bit) to your Bountysource once it's made.

**Comment by MajorTom87** [comment]
> I'm going to spend a little time working on FFV and try to fix the bugs and then FF Dimensions.

have you abandoned the ff4 the after years project?

**Comment by DaftOtter** [comment]
> Final Fantasy VI and VII are portable. VIII and IX are Unity so VERY UNLIKELY for someone to port them (unless a very huge bounty gets set for them).

Just out of interest, once one unity game gets ported does that mean it’s easier to port other unity games for anyone who might try? Or does each game require a huge amount of brand new work?

In your honest opinion, how much would a bounty have to be to port a unity game? Say one of the final fantasy pixel remasters, if I was going to bounty them I’d do them in order so let’s say the first one. How much would you or someone else require to put the work in? Or is it more of see what people are willing to pay then you decide?

Thanks for your time…

**Comment by DaftOtter** [comment]
> I'm going to spend a little time working on FFV and try to fix the bugs and then FF Dimensions.

Do you require a bounty to do this work? If not that’ll give me time to save up for VI. 

Thanks for all your hard work. I love your V port.

**Comment by Rinnegatamante** [comment]
> > Final Fantasy VI and VII are portable. VIII and IX are Unity so VERY UNLIKELY for someone to port them (unless a very huge bounty gets set for them).
> 
> Just out of interest, once one unity game gets ported does that mean it’s easier to port other unity games for anyone who might try? Or does each game require a huge amount of brand new work?
> 
> In your honest opinion, how much would a bounty have to be to port a unity game? Say one of the final fantasy pixel remasters, if I was going to bounty them I’d do them in order so let’s say the first one. How much would you or someone else require to put the work in? Or is it more of see what people are willing to pay then you decide?
> 
> Thanks for your time…

Unity games can already be ported "sorta" by decompiling the game and recompiling it after fixing all the issues. This is however something pretty illegal. (For reference, the "so said" Hollow Knight port is one of these).
In order to port an Android Unity game, a HUGE amount of work must be put and it absolutely doesn't mean said port can enable drag'n'drop execution of other Unity games (unlike Yoyo Loader) for a few reasons:
1) Unity projects can use or not il2cpp, this will largely change how the game runs internally.
2) Unity exported projects are completely symbol stripped, meaning you can't produce address independant patches. This means every single game (and every single different game version of the same game) will require different patches in order to run.
3) Not every Unity game runs on the same version of Unity engine meaning likely you'll need to produce different patches for different games.

Unless a very huge bounty is set up, I doubt anyone will ever care about looking into an Unity game since Unity in general is a bloatware (produced executables run bad on average due to overhead of the engine which doesn't fit well with Vita hardware starting with) and due to lack of symbols, working on it is a nightmare.

**Comment by DaftOtter** [comment]
> > > Final Fantasy VI and VII are portable. VIII and IX are Unity so VERY UNLIKELY for someone to port them (unless a very huge bounty gets set for them).
> > 
> > 
> > Just out of interest, once one unity game gets ported does that mean it’s easier to port other unity games for anyone who might try? Or does each game require a huge amount of brand new work?
> > In your honest opinion, how much would a bounty have to be to port a unity game? Say one of the final fantasy pixel remasters, if I was going to bounty them I’d do them in order so let’s say the first one. How much would you or someone else require to put the work in? Or is it more of see what people are willing to pay then you decide?
> > Thanks for your time…
> 
> Unity games can already be ported "sorta" by decompiling the game and recompiling it after fixing all the issues. This is however something pretty illegal. (For reference, the "so said" Hollow Knight port is one of these). In order to port an Android Unity game, a HUGE amount of work must be put and it absolutely doesn't mean said port can enable drag'n'drop execution of other Unity games (unlike Yoyo Loader) for a few reasons:
> 
> 1. Unity projects can use or not il2cpp, this will largely change how the game runs internally.
> 2. Unity exported projects are completely symbol stripped, meaning you can't produce address independant patches. This means every single game (and every single different game version of the same game) will require different patches in order to run.
> 3. Not every Unity game runs on the same version of Unity engine meaning likely you'll need to produce different patches for different games.
> 
> Unless a very huge bounty is set up, I doubt anyone will ever care about looking into an Unity game since Unity in general is a bloatware (produced executables run bad on average due to overhead of the engine which doesn't fit well with Vita hardware starting with) and due to lack of symbols, working on it is a nightmare.

Okay fairplay, so unless there’s a diehard final fantasy fan that also loves to port games to vita, I shouldn’t expect, even with a decent bounty, that it’ll ever happen.

In that case, I’ll save up whilst frangarcj is working on a patch for final fantasy v, then put a bounty on for VI after he’s released it. I may as well after vi is done get vii done as well. I just love my final fantasy with an absolute passion. I’ve got every single version of every final fantasy that’s playable on a hacked vita. I bought one of the FFX vita consoles, which I plan to turn into the ultimate final fantasy machine, lol.

Would you, if you are inclined to do so, make a short list of every portable android final fantasy, so basically the non-unity ones? I will do my absolute best to provide someone with a bit of pocket money for porting them. I just want every final fantasy that’s possible, and I’ll happily pay. To be honest just getting V patched and a good port of VI would be incredible for me. Anyway, enough of me ranting on.

Thank you for all the love for vita, you’ve got more respect for that hardware than Sony ever did, and thanks for taking the time to give a detailed response.

✌️
